cmd/cli/main.go

```diff
@@ -102,144 +102,125 @@ func main() {
 					},
 				},
 				Action: func(ctx *cli.Context) error {
-					// TODO: move this action's body to a separate function to decrease the
-					// code nesting significantly.
-
 					pathToFile := ctx.Args().Get(0)
 					if pathToFile == "" {
 						return fmt.Errorf("path to cairo file not set")
 					}
+
+					var hints map[uint64][]hinter.Hinter
+					var program *runner.Program
```

Comment:
Is there any reason to define the variables here now that you've removed the if statement below? I think in Go, it's preferred defining the variables at the point of initialization, unless you need them to be accessible outside the specific block, or if you need default values before conditional initialization, and such cases

Reply:
Yeah thats right